Tests/IntegrationTests/Roslyn/Resources/PluginInspector.java

```diff
@@ -63,21 +67,72 @@ public static void main(String[] args) throws Exception {
             System.exit(ErrorCode);
         }
 
-        Document doc = ProcessJar(jarPath);
+        String xmlFilePath = args.length == 2 ? args[1] : jarPath + ".dump.xml";
+
+        Integer exitCode = CreateAndInvokeInspector(jarPath, xmlFilePath);
+
+        System.exit(exitCode.intValue());
+    }
+
+    public static Integer CreateAndInvokeInspector(String jarPath, String xmlFilePath) throws MalformedURLException, ClassNotFoundException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
```

Comment:
The plugin inspector is Java code that instantiates a plugin, creates the extensions and dumps information about the extensions to an XML file. I haven't changed the XML format. However, I needed to change the way the inspector loaded plugins due to differences in the way the platform RulesXmlLoader and SqaleXmlLoader classes locate the resource files to load.
Previously, the inspector could use a simple UrlClassLoader, which was sufficient to locate the rules xml file. Unfortunately, we need something more complicated to correctly locate the sqale file resource. The comments in the class provide more info: we can discuss offline if necessary.